draft-gpew-priv-ppm.md

```diff
@@ -467,19 +468,27 @@ number generator. Each task has the following parameters associated with it:
   leader's endpoint MUST be the first in the list. The order of the
   `encrypted_input_shares` in a `Report` (see {{uploading-reports}}) MUST be the
   same as the order in which aggregators appear in this list.
-* `collector_config`: The HPKE configuration of the collector (described in
-  {{key-config}}). Having participants agree on this absolves collectors of the
-  burden of operating an HTTP server. See
-  [#102](https://github.com/abetterinternet/prio-documents/issues/102) for
-  discussion.
 * `max_batch_lifetime`: The maximum number of times a batch of reports may be
   used in collect requests.
 * `min_batch_size`: The minimum number of reports that appear in a batch.
 * `min_batch_duration`: The minimum time difference between the oldest and
   newest report in a batch. This defines the boundaries with which the batch
   interval of each collect request must be aligned. (See
   {{batch-parameter-validation}}.)
-* `protocol`: named parameter identifying the VDAF scheme in use.
+* A unique identifier for the VDAF being used for the task.
```

Comment:
I think we need a bit more than this. Suppose that a PPM task executes [`Prio3Aes128Histogram`](https://github.com/divviup/libprio-rs/blob/1bd54185d01110ff4486b8d71ed8f17f1ea77b78/src/vdaf/prio3.rs#L123). Besides that label, everyone also needs to know what the bucket boundaries are. Perhaps the PPM text should refer to VDAF's description of a [`Measurement` type](https://github.com/cjpatton/vdaf/blob/6904d4a6295da82daf7ecd09c094326db4fb8ccd/draft-patton-cfrg-vdaf.md?plain=1#L404)?

Reply:
I think this needs to be "something that fully defines the VDAF instance you're using, modulo the parameters generated from VDAF.setup() (since those change per invocation)," so I elaborated on this a bit.